src/workflows/client/client.py

```diff
@@ -0,0 +1,267 @@
+import httpx
+import time
+import json
+
+from typing import Literal, Any, Union, AsyncGenerator, AsyncIterator
+from contextlib import asynccontextmanager
+from logging import getLogger
+from workflows.events import StartEvent, Event
+from workflows import Context
+
+
+logger = getLogger(__name__)
+
+
+class WorkflowClient:
```

Comment:
I don't see a `/handlers` method added here. Should we include that? Related to the `/result` api, it returns a list of the same data model